app/_src/production/install-kumactl.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Install kumactl
+content_type: how-to
+---
+
+This how-to guide explains how to install `kumactl` in your environment.
+
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can do the following:
+
+* Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. 
+* Read and create resources in {{site.mesh_product_name}} in Universal mode.
+
+The `kumactl` binary is a client to the {{site.mesh_product_name}} HTTP API. 
+
+{% tabs install-kumactl useUrlFragment=false %}
+{% tab install-kumactl Kubernetes %}
+1. Download {{site.mesh_product_name}}:
+{% tabs install_kumactl useUrlFragment=false %}
+{% tab install_kumactl Script %}
+
+Run the following script to automatically detect the operating system and download Kuma:
+
+<div class="language-sh">
+  <pre class="no-line-numbers"><code>curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -</code></pre>
+</div>
+
+You can omit the `VERSION` variable to install the latest version.
+{% endtab %}
+{% tab install_kumactl Direct Link %}
+
+Download the distribution manually. Download a distribution for the **client host** from where you will be executing the commands to access Kubernetes:
+
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-centos-amd64.tar.gz">CentOS</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-rhel-amd64.tar.gz">RedHat</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-debian-amd64.tar.gz">Debian</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-ubuntu-amd64.tar.gz">Ubuntu</a>
+* <a href="https://download.konghq.com/mesh-alpine/kuma-{{ page.latest_version }}-darwin-amd64.tar.gz">macOS</a> or run `brew install kumactl`
+
+and extract the archive with `tar xvzf kuma-{{ page.latest_version }}.tar.gz`
+{% endtab %}
+{% endtabs %}
+
+1. Add the `kumactl` executable to your path:
+```
+cd kuma-{{ page.latest_version }}/bin
+PATH=$(pwd):$PATH
+```
+
+{% endtab %}
+{% tab install-kumactl Helm %}
+Add the {{site.mesh_product_name}} charts repository:
+```sh
+helm repo add kuma https://kumahq.github.io/charts
+helm upgrade -i kuma kuma/kuma
+```
+
+{% endtab %}
+{% tab install-kumactl OpenShift %}
+Download {{site.mesh_product_name}}:
+```sh
+curl -L https://kuma.io/installer.sh | VERSION={{ page.latest_version }} sh -
+```
+{% endtab %}
+{% tab install-kumactl Docker %}
+Download {{site.mesh_product_name}}:
+```sh
+docker pull docker.io/kumahq/kuma-cp:{{ page.latest_version }}
+docker pull docker.io/kumahq/kuma-dp:{{ page.latest_version }}
```

Comment:
why do we need cp and dp?

Reply:
also for docker I guess an alias like this:

```
alias kumactl="docker run --rm -it kumahq/kumactl:{{ page.latest_version }} kumactl"
```

would be nice

Reply:
> why do we need cp and dp?

I can move these to a different page, I just didn't see them in any other doc. Do you know if we have these written down on another page? If not, what page would these make the most sense on?

Reply:
> I can move these to a different page, I just didn't see them in any other doc.

`kuma-cp` and `kuma-dp` are not required to get `kumactl` running. I think we could just remove the docker pull commands for these two.